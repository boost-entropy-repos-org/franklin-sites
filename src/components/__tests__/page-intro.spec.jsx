import React from 'react';
import renderer from 'react-test-renderer';
import PageIntro from '../page-intro';

describe('PageIntro component', () => {
  test('should render', () => {
    const count = 1000;

    const component = renderer
      .create(
        <PageIntro title="Title" resultsCount={count}>
          <div>Some content</div>
        </PageIntro>,
      )
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
