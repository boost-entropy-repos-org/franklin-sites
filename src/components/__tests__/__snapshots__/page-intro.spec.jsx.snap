// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageIntro component should render 1`] = `
<div
  className="page-intro"
>
  <h2>
    <button
      className="dropdown"
      onClick={[Function]}
      type="button"
    >
      <test-file-stub />
       
      Title
    </button>
    <small>
       
      1,000
       
      results
    </small>
  </h2>
  <div
    className="intro-content"
    style={
      Object {
        "display": "none",
      }
    }
  >
    <div>
      Some content
    </div>
    <div
      className="intro-links"
    />
  </div>
</div>
`;
